inc/namespace.php

```diff
@@ -3,7 +3,7 @@
 Plugin Name: WP Simple SAML
 Description: Integrate SAML 2.0 IDP without the hassle
 Author: Shady Sharaf, Human Made
-Version: 0.1
+Version: 0.6.0
```

Comment:
Why the very large version update? The version in master is 0.4.1, why the jump?

Reply:
The latest release tag was 0.5.1, this introduces a new action so I opted for a minor release. Semantic versioning of a plugin below 0.x allows for this.

Reply:
This file now says 0.6.0 (which is a major release under 0.x), but the _actual_ plugin version is in `plugin.php` not this file, which is 0.4.1: https://github.com/humanmade/wp-simple-saml/blob/83961f37e11b358a1d3a0321c5a52386a60c40fe/plugin.php#L6

Seems like we need to remove it from this file, fix the main one, and make sure we keep those in sync.